### PR TITLE
Initialize `Config()` as a module attribute

### PR DESCRIPTION
**Pull request by stefsmeets**
This PR sets the config as a module attibute.

Instead of:

```python
from duqtools.config import Config as cfg
cfg().variable
```

use

```python
from duqtools.config import cfg
cfg.variable
```

### CODE REVIEW

**Comment by stefsmeets** [comment]
Hi @v1kko I changed the config to be initialized as a module attribute, so that we don't have to initialize it every time and we catch any errors in the config module, not the first time it's called. Would you be OK with this?

**Comment by v1kko** [comment]
Yes I also tried this, but I found that  the module variable was copied, not referenced, thus the program only had access to the "default" cfg, not the one we initialize with the yaml file.

However if it works like this, then I absolutely agree this is better.

PS: Config is currently not be reinitialized every time, since __init__ is overloaded and does nothing with no arguments

File: duqtools/config.py [hunk 1]
```diff
@@ -162,3 +162,6 @@ def __new__(cls, *args, **kwargs):
         if not Config._instance:
             Config._instance = object.__new__(cls)
         return Config._instance
+
+
+cfg = Config()
```
Comment (v1kko) [inline duqtools/config.py:167]: if the `from duqtools.config import cfg` does copy the cfg object, then we could also turn cfg into a property:

```suggestion
@property
def cfg():
    return Config()
```

**Comment by v1kko** [comment]
Works for me now, perfect!

**Pull request merged**